database/mock_data/PopulateDB.sql

```diff
@@ -151,6 +151,7 @@ insert into member (member_type_id, first_name, last_name, phone_number, occupat
 insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('6d817fa9-dca1-4f95-bd0a-e3c5c52d5639', 1, 'Elijah', 'Parrish', '315-396-2469', 'Senior Software Dev', 'elijah.parrish321@gmail.com', '2005-12-11', '2018-05-19', 1);
 insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('12c', 2, 'Patin', 'Lelliott', '913-810-0953', 'Senior Financial Analyst', 'plelliott2q@multiply.com', '2005-12-11', '2018-05-19', 1);
 insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('6e99b3fd-5771-4484-804c-2a42f807e365', 1, 'CJ', 'Schaefer', '216-867-5309', 'Senior Software Dev', 'not.cjs.email@gmail.com', '2005-12-11', '2018-05-19', 1);
+insert into member (uuid, member_type_id, first_name, last_name, phone_number, occupation, email, birthdate, date_joined, active) values ('a6f6b09d-83fd-414d-99a4-71ddffde40e0', 1, 'Brianna', 'Fehring', '513-929-5010', 'Senior Software Dev', 'bnf8555@rit.edu', '1997-12-05', '2018-05-19', 1);
```

Comment:
In `src/__tests__/integration/member.test.ts`:
- At line 39, change `[103, 104]` to `[104, 105]`
- At line 57, change `15` to `16` (this will allow the second failing integration test to pass)